manim/__main__.py

```diff
@@ -19,8 +19,8 @@
 
 
 def open_file_if_needed(file_writer):
+    curr_stdout = sys.stdout
     if config["verbosity"] != "DEBUG":
```

Comment:
Can you explain what does it change doing this here?

Reply:
As explained in the commit, rebinding this name here has essentially no performance impact and reduces the number of warnings thrown by pylance

Reply:
What warning did it throws?

Reply:
'Possibly Unbound Here'
As a human I can see that is isn't (as `config['verbosity']` should remain constant throughout the block) but there's no performance impact to this fix